base/setup/usetup/lang/ru-RU.h

```diff
@@ -12,37 +12,37 @@ static MUI_ENTRY ruRULanguagePageEntries[] =
     {
         4,
         3,
-        " ��⠭���� ReactOS " KERNEL_VERSION_STR,
```

Comment:
Hi, what has been changed in this file precisely? I fail to see any visual differences...

Reply:
Encoding, probably. He used GitHub online source editor.

Reply:
If so, that's definitively the wrong way to proceed.

Reply:
@binarymaster You're right.

Reply:
@HBelusca Ok, I will consider.